.circleci/config.yml

```diff
@@ -116,7 +116,7 @@ jobs:
             export CCACHE_DIR=$(realpath .ccache)
             ccache -sz -M 5Gi
             source /opt/rh/gcc-toolset-9/enable
-            make debug NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=16
+            make debug NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=4
```

Comment:
`8` should work?
We have to make this change to the `linux-parquet-s3-build` as well.

Reply:
Thanks for checking!